docs/system/system-metrics.md

```diff
@@ -260,9 +459,13 @@ This metric is [recommended][MetricRecommended].
 |---|---|---|
 | `used` | used | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `free` | free | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `shared` | shared | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Removed, report shared memory usage with `metric.system.memory.shared` metric |
+| `shared` | shared |  ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Removed, report shared memory usage with `metric.system.memory.shared` metric |
```

Comment:
super-nit: extra space

Reply:
You have no idea how many hours I've spend trying to control whitespace in JINJA... I'll add this to my never-decreasing list.

Reply:
Should be fixed